akka-http/build.sbt

```diff
@@ -2,7 +2,6 @@ import Dependencies._
 
 libraryDependencies ++= Seq(
```

Comment:
I think we are missing two explicit dependencies here: `com.typesafe.config` and `com.typesafe.akka.akka-http-core`.

Reply:
Yes. I think that was due to a merge gone wrong.